src/jobs/weightDriftAuditJob.ts

```
@@ -0,0 +1,230 @@
+ * - Audit log entries created for all policy drift events
+ */
+
+import { logger } from "../config/logger";
+import { weightDriftAuditService } from "../services/reserve/WeightDriftAuditService";
+import { sendEmail } from "../services/notification";
+import { config } from "../config/env";
+
+const DEFAULT_INTERVAL_DAYS = 7; // Run weekly
+const INTERVAL_MS =
+  (parseInt(
+    process.env.WEIGHT_DRIFT_AUDIT_INTERVAL_DAYS || String(DEFAULT_INTERVAL_DAYS),
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)